src/pages/Notifications/BehaviorGroupWizard/Steps/BasicInformationStep.tsx

```diff
@@ -28,7 +28,7 @@ const BasicInformationStep: React.FunctionComponent = () => {
 };
 
 export const schema = Yup.object({
-    displayName: Yup.string().min(1).required('Behavior group name is required')
+    displayName: Yup.string().min(1).max(150).required('Behavior group name is required')
```

Comment:
@mmenestr What error should be displayed if the name is over 150 characters?

Reply:
I would display help text under the input with "Must be 150 characters or less." and if they exceed it, it turns red and the input takes on an error state
![Uploading image.png…]()

like this but with a different message obviously^

Reply:
@Mishrasubha could you add the message as above?

Should be something like:

```suggestion
    displayName: Yup.string().min(1).max(150, 'Must be 150 characters or less').required('Behavior group name is required')
```

Reply:
On it